example/README.md

```diff
@@ -0,0 +1,28 @@
+# Example
+
+The example instrumenting simple HTTP server and client application.
+
+The applications are setup to send spans to OTel Collector
+which propagates them to both Splunk Observability Cloud
+and a local Jaeger.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+## Usage
+
+```sh
+SPLUNK_ACCESS_TOKEN=secret ./run.sh
+```
+
+The value for `SPLUNK_ACCESS_TOKEN` can be found
+[here](https://app.signalfx.com/o11y/#/organization/current?selectedKeyValue=sf_section:accesstokens).
```

Comment:
To obtain an access token, see [Create and manage user API access tokens using Splunk Observability Cloud](https://docs.splunk.com/Observability/admin/authentication-tokens/api-access-tokens.html#admin-api-access-tokens).